packages/wdio-devtools-service/src/types.ts

```diff
@@ -25,7 +25,11 @@ export interface CoverageReporterOptions {
     /**
      * Options for coverage report
      */
-    options?: any
+    options?: any,
+    /**
+     * Exclude code coverage files
+     */
+     exclude?: any[];
```

Comment:
Let's type it properly:
```suggestion
     * Exclude code coverage files
     * @default []
     */
    exclude?: (RegExp | string)[]
```